tests/test_models.py

```diff
@@ -29,3 +31,13 @@ def test_daily_mean_integers():
     # Need to use Numpy testing functions to compare arrays
     npt.assert_array_equal(daily_mean(test_input), test_result)
 
+
+@pytest.mark.parametrize('data, expected_standard_deviation', [
```

Comment:
I came up with some additional tests you could add:
- do all if statements evaluate as true or false
- would the data allow negatives?

Reply:
Added a test for negative data values in commit  b0c7a3c